resolvers.ts

```diff
@@ -459,6 +460,45 @@ export const resolvers: Resolvers = {
         })
       );
     },
+    WorkflowRunsAggregate: async (root, args, context, info) => {
+      const input = args.input;
+      
+      const { projects } = await get("/projects.json" + 
+        formatUrlParams({
+          projectId: input?.todoRemove?.projectId,
+          domain: input?.todoRemove?.domain,
+          limit: TEN_MILLION, // do we need pagination? or single request?
```

Comment:
maybe we can paginate if we see some performance issues? Otherwise I'm fine trying out all in one go for now!